src/helpers/language.ts

```diff
@@ -1,17 +1,44 @@
 ﻿import MyDataHelps from "@careevolution/mydatahelps-js"
 import englishStrings from "./strings-en"
 import spanishStrings from "./strings-es"
+import dutchStrings from "./strings-nl"
+import germanStrings from "./strings-de"
+import frenchStrings from "./strings-fr"
+import portugueseStrings from "./strings-pt"
+import italianStrings from "./strings-it"
+import polishStrings from "./strings-pl"
+
+export type Language = "" | "en" | "es" | "nl" | "de" | "fr" | "pt" | "it" | "pl"
 
 export function language(key: string) {
-	if (!MyDataHelps.getCurrentLanguage().toLowerCase().startsWith("es")) {
-		return englishStrings[key];
-	} else {
-		if (spanishStrings[key] != null) {
-			return spanishStrings[key];
-		} else {
-			return englishStrings[key];
-		}
-	}
+	const currentLanguage: Language = getLanguageFromIso(MyDataHelps.getCurrentLanguage());
+
+	if (currentLanguage == "en") return englishStrings[key];
```

Comment:
I think we want to fall back to english if the individual string isn't defined in the target language; see the previous logic where it actually checked to see if the string was there...so it should probably be something like:
```
let resolvedString = null;
if(currentLanguage == "es") { resolvedString = spanishStrings[key]; }
//etc

if(!resolvedString){
   resolvedString = englishStrings[key];
}
return resolvedString;
```